dev-util/xmake/xmake-2.8.5.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A cross-platform build utility based on Lua"
+DESCRIPTION="What is Xmake? \
+1. Xmake is a cross-platform build utility based on the Lua scripting language. \
+2. Xmake is very lightweight and has no dependencies outside of the standard library. \
+3. Uses the \`xmake.lua\` file to maintain project builds with a simple and readable syntax. \
+Xmake can be used to directly build source code (like with Make or Ninja), or it can generate project source files like CMake or Meson. It also has a built-in package management system to help users integrate C/C++ dependencies."
+HOMEPAGE="https://xmake.io/"
+COPYRIGHT="2015-present TBOOX Open Source Group"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/xmake-io/xmake/releases/download/v$portVersion/xmake-v$portVersion.tar.gz"
+CHECKSUM_SHA256="19c67be3bcfcbda96ca87a18effac39bfccc443fbb3754c7bbc01511decd24af"
+PATCHES="xmake-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+
+PROVIDES="
+	xmake = $portVersion
+	cmd:xmake = $portVersion
+	cmd:xrepo = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libatomic
+	lib:libncurses >= 6
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libncurses >= 6
```

Comment:
This should probably have a `devel:libatomic` build requirement to match the `lib:libatomic` even though the package will probably be active anyhow.